.github/workflows/lint.yaml

```diff
@@ -0,0 +1,48 @@
+name: Node.js CI lint/codecov/commit check
+
+on:
```

Comment:
If you're moving away from jenkins, you might want to run this on pull requests as well:
```yaml
on:
  push:
    branches:
    - master
  pull_request:
    branches:
    - master
```

Reply:
@andersfylling  The idea was that this should _only_ run on pull requests, I have a 3rd workflow that should run on master pushes. Fixed. (and it's moving away from Travis, not jenkins)